src/qt/addresstablemodel.cpp

```diff
@@ -341,7 +341,7 @@ void AddressTableModel::updateEntry(const QString &address,
     priv->updateEntry(address, label, isMine, purpose, status);
 }
 
-QString AddressTableModel::addRow(const QString &type, const QString &label, const QString &address)
+QString AddressTableModel::addRow(const QString &type, const QString &label, const QString &address, const bool bech32)
```

Comment:
Could receive `OutputType type` instead of `bool bech32`? Just pass that to `AddDestinationForKey` below.

Reply:
I think a boolean makes more sense here, because the check-box can only tell that `OUTPUT_TYPE_BECH32` should be used. If it's unchecked,  it doesn't know if the fallback is `OUTPUT_TYPE_P2SH` or `OUTPUT_TYPE_LEGACY`.

Reply:
Update: it now uses `OutputType`.